### ISSUE TITLE
Make jiant pip installable

### ISSUE THREAD

**Issue by sleepinyourhat** [nyu-mll/jiant#877]
This is necessary for Flambé support, and should be relatively painless.

**Comment by davidbenton** [comment]
I can work on a PR. Looks like this is tagged for a release, but I'm not sure of the release strategy. Should the PR be to master or to a (new?) release branch with v1.0.1 as the parent?

**Comment by pruksmhc** [comment]
We've been just pushing to master as of now, although we should probably start making release branches for best practices

**Comment by sleepinyourhat** [comment]
Yep—a PR directly to master would work fine for now, if you're interested. Thanks!

**Comment by davidbenton** [comment]
Sounds good, will do!
